Makefile

```diff
@@ -169,20 +169,21 @@ Tools/aiger-ltl-model-checker:
 	cd Tools ; git clone https://github.com/reactive-systems/aiger-ltl-model-checker.git
 
 # cryptominisat
-Tools/cryptominisat5: Tools/cryptominisat-5.6.8/build
-	cp Tools/cryptominisat-5.6.8/build/cryptominisat5_simple Tools/cryptominisat5
+cms_version=5.7.0
```

Comment:
Using variables is good.  
Could you also do this for the other tools in a different commit?

Reply:
Already done